shared/featureconfig/deprecated_flags.go

```diff
@@ -0,0 +1,19 @@
+package featureconfig
+
+import "github.com/urfave/cli/v2"
+
+// Deprecated flags list.
+const deprecatedUsage = "DEPRECATED. DO NOT USE."
+
+var (
+	// To deprecate a feature flag, first copy the example below, then insert deprecated flag in `deprecatedFlags`.
+	exampleDeprecatedFeatureFlag = &cli.StringFlag{
+		Name:   "name",
+		Usage:  deprecatedUsage,
+		Hidden: true,
+	}
+)
+
+var deprecatedFlags = []cli.Flag{
+	exampleDeprecatedFeatureFlag,
```

Comment:
lets not include this in the default deprecated set.

Reply:
This is included for the test. We could remove it and the test all at once but it would be nice to have a test in place to guard against proper format